telemetry-attribute-utils/src/main/java/org/hypertrace/telemetry/attribute/utils/messaging/MessagingTelemetryAttributeUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.hypertrace.telemetry.attribute.utils.messaging;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.shared.SpanAttributeUtils;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.span.constants.v1.RabbitMq;
+
+/**
+ * Utility class to fetch messaging system span attributes
+ */
+public class MessagingTelemetryAttributeUtils {
+
+  private static final List<String> RABBITMQ_ROUTING_KEYS =
+      new ArrayList<>(Arrays.asList(
+          RawSpanConstants.getValue(RabbitMq.RABBIT_MQ_ROUTING_KEY),
+          OtelMessagingAttributes.RABBITMQ_ROUTING_KEY.getValue()
```

Comment:
Shouldn't we have to fallback `messaging.system` (https://github.com/open-telemetry/opentelemetry-specification/blob/master/semantic_conventions/trace/messaging.yaml) derived from?

Reply:
Aren't they same keys right now - `RabbitMq.RABBIT_MQ_ROUTING_KEY, OtelMessagingAttributes.RABBITMQ_ROUTING_KEY.getValue()` ?

Reply:
Yes they are same but doesn't matter. Tomorrow if the value changes, the code doesn't have to change. 
If you see rabbitmq resolver the only thing we fetch is routing key, so falling back to system won't help.